minutes/2016-06-03.md

```diff
@@ -0,0 +1,248 @@
+## 2016-06-03
+
+Time: 14:00 UTC
+
+Hangout link: [](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)[https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)
+
+**Attendees**
+
+Ray, Matt, Jonathan, Phil, Jonas, Michael, Philippe, John, [Bj�rn Gr�ning](https://conda-forge.hackpad.com/ep/profile/DMmBLyb21HK), Jan
```

Comment:
I think I am missing Jonas' GitHub handle. So, if someone would like to ping him, that would be nice.